Package.swift

```diff
@@ -0,0 +1,9 @@
+import PackageDescription
+
+let package = Package(
+    name: "GRDB.swift",
+    dependencies: [
+        .Package(url: "https://github.com/groue/SQLiteSDK.git", majorVersion: 0)
```

Comment:
Is the convention of prefixing C packages with the letter C dead? The [SPM documentation](https://github.com/apple/swift-package-manager/blob/master/Documentation/Usage.md#require-system-libraries) still uses it: Clibgit, CJPEG...